source/_components/sabnzbd.markdown

```diff
@@ -64,6 +64,11 @@ ssl:
   required: false
   default: false
   type: boolean
+urlbase:
+  description: The base URL SABnzbd is running under.
+  required: false
+  default: /
```

Comment:
https://github.com/home-assistant/home-assistant/pull/21792/files#diff-a6557fb9357fc3d47af2117456677a23R35 I see a different default?

Reply:
I was trying to remain consistent with how Radarr and Sonarr define and implement this:

https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/radarr.py#L29

Their documentation claims a default of '/' which is the effective default.